utils/docker/Dockerfile.bionic

```diff
@@ -137,6 +24,7 @@ RUN apt-get update && apt-get install -y python3.8 python3-pip && \
 # === BAKE BROWSERS INTO IMAGE ===
 
 ENV PLAYWRIGHT_BROWSERS_PATH=/ms-playwright
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
shall we use [`ARG` instead](https://vsupalov.com/docker-arg-vs-env/)?

Reply:
done